app/Resources/doc/import.md

```diff
@@ -10,40 +10,25 @@ Map Building
 Build map via file upload
 -------------------------
 
-The most common way to build a map will likely be uploading a sample file. The system will read the header column
-row and build an initial map. It will try to match any column name in the file with database columns.
+The most common way to build a map will likely be uploading a sample file. The system will read the header column row and build an initial map. It will try to match any column name in the file with database columns.
 
```

Comment:
Huh, okay. What's the target line
      length? Is it because it makes viewing diffs difficult? Should I
      leave it the way it is now? Because I can fix it easily in
      Sublime.
      
    
    
      
        
          
            
               Jim Grusendorf
                President
                Keen Mouse Productions
                  Inc.
                jgrusendorf@keenmouse.com
              
            
                
          
        
      
    
    On 2016-04-26 14:23, Nathanael Noblet
      wrote:
    
    
      In app/Resources/doc/import.md:
      > @@ -10,40 +10,25 @@ Map Building
>  Build map via file upload
>  -------------------------
>  
> -The most common way to build a map will likely be uploading a sample file. The system will read the header column
> -row and build an initial map. It will try to match any column name in the file with database columns.
> +The most common way to build a map will likely be uploading a sample file. The system will read the header column row and build an initial map. It will try to match any column name in the file with database columns.
>  

      Its better if things have newlines so they are readable when
        not viewed via a formatter / HTML like on github the lines wrap
        in a simple text editor.
      —
        You are receiving this because you authored the thread.
        Reply to this email directly or view
          it on GitHub